packages/rancher-monitoring/generated-changes/patch/values.yaml.patch

```diff
@@ -260,6 +275,38 @@
 +      - effect: "NoSchedule"
 +        operator: "Exists"
 +
++rke2IngressNginx:
++  enabled: false
++  metricsPort: 10254
++  component: ingress-nginx
++  clients:
++    port: 10015
++    useLocalhost: true
++    tolerations:
++      - effect: "NoExecute"
++        operator: "Exists"
++      - effect: "NoSchedule"
++        operator: "Exists"
++    # in the RKE2 cluster, the ingress-nginx-controller is deployed as
++    # a Deployment with 1 pod when RKE2 version is <= 1.20,
++    # a DaemonSet when RKE2 version is >= 1.21
++    deployment:
++      enabled: false
```

Comment:
For future reference / other reviewers: https://github.com/rancher/dashboard/issues/3189

@jiaqiluo we should also figure out if we should release note this requirement or add it somewhere to our docs.

Reply:
we can add it to both the release note and the documentation.  I will follow up on it.

Reply:
https://github.com/rancher/docs/issues/3326
https://github.com/rancher/dashboard/issues/3201